packages/core/src/application.ts

```diff
@@ -135,6 +135,9 @@ export class Application extends Context {
     request: ServerRequest,
     response: ServerResponse,
   ) {
+    // allow CORS support for all endpoints so that users
+    // can test with online SwaggerUI instance
+    response.setHeader('Access-Control-Allow-Origin', '*');
```

Comment:
I get we need to set the Header for all endpoints for Swagger but I'm not sure if users would want that enabled in production, especially with the `*`. Can this be configurable / toggleable via options (ideally ENV variable)? / Can we look into (perhaps as a separate issue) look into turning this into a configurable component?

Reply:
We should follow the defaults described by http://loopback.io/doc/en/lb3/middleware.json.html#cors-settings.
```
Access-Control-Allow-Origin: <Origin request header>
Access-Control-Allow-Credentials: true
Access-Control-Allow-Max-Age: 86400
```

Please note CORS is only enforced by browsers and it's not a security concern for the server itself.

Reply:
@virkt25 I think for GA we are aiming to do that (see #488), so I'll leave it as is for now.